cfme/tests/cloud/test_providers.py

```diff
@@ -303,6 +305,37 @@ def test_openstack_provider_has_api_version():
     assert view.api_version.is_displayed, "API version select is not visible"
 
 
+@pytest.mark.tier(3)
+@pytest.mark.usefixtures('has_no_cloud_providers')
```

Comment:
**QUESTION**
Why is `has_no_cloud_providers` not in the test arguments?
```python
def test_select_key_pair_none_while_provisioning(request, has_no_cloud_providers):
```

Reply:
hm, not a good reason here, yup better to move it